python/src/aiconfig/Config.py

```diff
@@ -53,6 +54,7 @@
     DefaultAnyscaleEndpointParser("AnyscaleEndpoint")
 )
 ModelParserRegistry.register_model_parser(GeminiModelParser("gemini-pro"), ["gemini-pro"])
+ModelParserRegistry.register_model_parser(ClaudeBedrockModelParser(), ["Claude"])
```

Comment:
Need to figure out model vs model provider but for now just registering under "Claude" as well